libcob/ChangeLog

```diff
@@ -43,6 +49,18 @@
 	* common.c (DllMain) [_MSC_VER]: added calls to _CrtSetReportMode
 	  to disable Windows error popups and redirect them to stderr
 
+2024-08-17  Ammar Almorsi <ammaralmorsi@gmail.com>
+
+	FR #474: add runtime configuration to hide cursor for extended screenio
+	* common.c, coblocal.h (cob_settings): added boolean COB_HIDE_CURSOR
+	* screenio.c (cob_settings_screenio): implemented runtime config to hide
+	  the cursor
+
+2024-08-04  David Declerck <david.declerck@ocamlpro.com>
+
+	Adjustments to merge 2020-06-20:
+	* common.c (cob_sys_x91): activated functions 46-49
```

Comment:
Do you remember if we added tests for this as well?

Reply:
Yes we did (reveision 5325, file run_extensions.at).
(note: here I just moved this entry as it was not at the right place)